posts/2017-05-10-serverless-v1.13.0.md

```diff
@@ -0,0 +1,66 @@
+---
+title: New in Serverless v1.13 - Use values of async functions in your Serverless variables
```

Comment:
What about something like: 

"Serverless v1.13 opens up world of configuration possibilities with new async variable support"